docs/index.md

```diff
@@ -0,0 +1,459 @@
+# Algorand transaction subscription / indexing
+
+## Quick start
+
+```{testcode}
```

Comment:
@PhearZero thats the example of a doctest using https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html. In case remaining docs need to use them, should be a matter of replacing `python` with `testcode` to make them runnable

Reply:
Let's sync on this! It also works in docstrings, here is an example that also comes in the intelisense :


![Demo-Doctest](https://github.com/user-attachments/assets/0482dd4e-bfc5-48df-b176-3cf995547181)